src/mpyl/project.py

```diff
@@ -190,17 +190,44 @@ def from_config(values: dict):
         return Probe(path=TargetProperty.from_config(values["path"]), values=values)
 
 
+@dataclass(frozen=True)
+class Alert:
+    name: str
+    expr: str
+    for_duration: str
+    description: str
+    severity: str
+
+    @staticmethod
+    def from_config(values: dict):
+        if not values:
+            return None
+        name = values.get("name")
+        expr = values.get("expr")
+        for_duration = values.get("forDuration")
+        description = values.get("description")
+        severity = values.get("severity")
+        if not name or not expr or not for_duration or not description or not severity:
+            return None
```

Comment:
Wouldn't you want to show some sort of warning if only part of the values are passed?